Enhancements + Travis.ci

bayleedev opened the pull request.

Commit:
Add comScore tracking

nwp committed:
Update tracking URI per most recent WT tag builder

nwp committed:
Update to properly deal ith boolean values in output

nwp committed:
Add option to disable noscript for WebTrends

nwp committed:
Add noscript option to comScore

nwp committed:
Add comments to methods

nwp committed:
Update to include Webtrends 10 and comScore

nwp committed:
Update to correctly name comScore element and treat account as string

Commit:
add quantcast

Commit:
Name should be adapter

Commit:
Have a _views array instead of _section string.

Commit:
Update trackers to load views not the single section and shells.

Commit:
Update chartbeat to support the new view format

Commit:
Add support for views in analytics

Commit:
Remove the current shells support and add views support.

Commit:
Remove unnecessary Shell namespaces

Commit:
Change all the current sections to views

Commit:
Add backwards compatibility for old rendering

Commit:
Remove unnecessary section method

Commit:
Change section to views in shell and update docblocks.

Commit:
Check if set not if string.

Commit:
Don't make special cases for Shell

Commit:
Remove element support

Commit:
Add view to the autoConfig array and remove section

Commit:
Update readme

Commit:
Add view rendering suppport.

Commit:
Update readme with new view support information.

Commit:
Should be calling it by key not name

Commit:
Pass all the tests.

Commit:
Add travis support

Commit:
Add travis image to readme

Commit:
Testing new travis file

Commit:
Travis support.

Commit:
Quantcast added in 20317787c6a72f41a1150ba2c138f1587fa3ce2f

joseym commented:
I don't see any updates to the readme, does the use of the plugin not change and/or does this not provide additional support that should be explained?

If so please update the documentation.

bayleedev commented:
Actually I see an issue with that. The only thing that might change is using "section" and "element" but I added backwards compatibility, however I see I only did that with shell(:52) and not tracker.

Commit:
Move compatibality from shell to tracker

Commit:
Add support single only section or only element

Commit:
Update docs to support views or section

Commit:
Update webtrends to support noscript in the body

bayleedev commented:
Moved the logic into Tracker and extended the logic.

Updated webtrends to put the noscript items at the end of the body.

Updated the readme to show how to overwrite specific views.
